domain/policy_test.go

```diff
@@ -181,6 +182,103 @@ func TestStep_ResolveApprovers(t *testing.T) {
 	}
 }
 
+func TestRequirementTrigger(t *testing.T) {
```

Comment:
Can you add a test case where condition and expression both are set ?

Reply:
updated